.gitignore

```diff
@@ -1,2 +1,4 @@
 /luacov.*
 /site/
+*.rbxmx
+*.rbxm
```

Comment:
Instead of ignoring _all_ model files in the repository, can we ignore just ones in the root?

```suggestion
/*.rbxmx
/*.rbxm
```

Reply:
I didn't seem to notice this, I'll commit this suggestion.